cmd/disk-cache-backend.go

```diff
@@ -175,8 +175,8 @@ func newDiskCache(ctx context.Context, dir string, config cache.Config) (*diskCa
 	}
 	go cache.purgeWait(ctx)
 	cache.diskSpaceAvailable(0) // update if cache usage is already high.
-	cache.NewNSLockFn = func(ctx context.Context, cachePath string) RWLocker {
-		return cache.nsMutex.NewNSLock(ctx, nil, cachePath, "")
+	cache.NewNSLockFn = func(cachePath string) RWLocker {
+		return cache.nsMutex.NewNSLock(nil, cachePath, "")
```

Comment:
Here it should new NewNSLock without the `nil`?

Reply:
Not sure I understand. TBH I don't understand the original code, but this should do the same AFAICT?

Reply:
What I am saying is that if you remove everywhere, not remove this for lsync as well @klauspost consistently changed across.

Reply:
Still not sure what you mean. The `nil` is a ` lockersFn func() []dsync.NetLocker` - it is not a context.